.gitignore

```diff
@@ -36,3 +36,16 @@ build/
 *.d
 utests*
 systest
+
+# CMake and Eclipse files
```

Comment:
I think we can scrap all our changes here and just remove all those files if present.

Reply:
Agreed - I'll take those back out

Reply:
The .gitignore changes are gone now.